src/miner.h

```diff
@@ -89,7 +89,7 @@ struct CompareTxIterByAncestorCount {
     {
         if (a->GetCountWithAncestors() != b->GetCountWithAncestors())
             return a->GetCountWithAncestors() < b->GetCountWithAncestors();
-        return CTxMemPool::CompareIteratorByHash()(a, b);
+        return CompareTxMemPoolEntryByEntryTime()(*a, *b);
```

Comment:
Is the transaction entry time is the same for different nodes? Does it cause different nodes have different order problem?

Reply:
It's only in miner though so doesn't make a difference